src/checkpoint1/checkpoint1.css

```diff
@@ -0,0 +1,218 @@
+body {
+    display: grid;
+    height:100%;
+    width:100%;
+    grid-template-columns: repeat(20,1fr);
+    grid-template-rows:repeat(20,1fr);
+}
+
+header {
+    background-color: rgb(83, 77, 77);
+    color:rgb(241, 236, 236);
+    text-align: center;
+    grid-column: 1 / 20;
+    grid-row: 1 / 2;
+    text-align: right;
+    padding: 15px;
+    word-spacing: 70px;
+}
+
+aside {
+    background-color: rgb(37, 210, 223);
+    text-align: center;
+    font-size: 15px;
+    color:white;
+    grid-column: 1/3;
+    grid-row: 2/20;
+    display:flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: space-around;
+    padding: 10px;
+}
+
+.title {
+    grid-column: 1/1;
+    grid-row: 1/1;
+    color:white;
+    font-size: 30px;
+    margin-left: 40px;
+    margin-top: 5px;
+}
+
+main {
+    background-color: lightgrey;
+    grid-column: 3/20;
+    grid-row: 2/20;
+}
+
+.dash {
+    font-size: 40px;
+    grid-column: 4/12;
+    grid-row: 2/2;
+    margin-top: 20px;
+    display: flex;
+
+}
+
+h6 {
+    font-size: 20px;
+    padding: 10px;
+    margin-top: 8px;
+    color: grey;
+
+}
+
+nav {
+    background-color: white;
+    grid-column: 4/19;
+    grid-row: 4/5;
+    word-spacing: 20px;
+    display: flex;
+}
+
+.icons {
+    background-color: white;
+    display: flex;
+    justify-content: space-around;
+}
+
+.one {
+    grid-column: 4/7;
+    grid-row: 6/10;
+    background-color: white;
+    display:flex;
+    flex-direction: column;
+    justify-content: space-around;
+}
+
+.two {
+    grid-column: 8/11;
+    grid-row: 6/10;
+    background-color: white;
+    display:flex;
+    flex-direction: column;
+    justify-content: space-around;
+}
+
+.three {
+    grid-column: 12/15;
+    grid-row: 6/10;
+    background-color: white;
+    display:flex;
+    flex-direction: column;
+    justify-content: space-around;
+}
+
+.four {
+    grid-column: 16/19;
+    grid-row: 6/10;
+    background-color: white;
+    display:flex;
+    flex-direction: column;
+    justify-content: space-around;
+}
+
+.five {
+    grid-column: 4/10;
+    grid-row: 11 / 19;
+    background-color: white;
+}
+
+.six {
+    grid-column: 11/19;
+    grid-row: 11/19;
+    background-color: white;
+}
+
+h3 {
+    font-size: 40px;
+    margin-top: 60px;
+    color: grey;
+}
+
+h4 {
+    font-size: 20px;
+    margin-top: 5px;
+    padding: 10px;
+    color: rgb(83, 139, 241);
+}
+
+p {
+    font-size: 20px;
+    font-weight: bold;
+    padding: 10px;
+}
+
+.img1 {
+    height:175px;
+    width: 320px;
+    padding: 20px;
+    background-size: cover;
+}
+
+.img2 {
+    height:175px;
+    width: 420px;
+    padding: 20px
+}
```

Comment:
with images, you only want to set either the height or the width. and you want them to be 100% the container they're in. remove: height, padding, background-size. change to `width: 100%;`. now for me, the second image is larger than the first and I don't understand why. but it works!